### PR TITLE
RSPY-491 + RSPY-492 + RSPY-495 + RSPY-506: Update CADIP/AUXIP helm configuration

### PR DESCRIPTION
**Pull request by vprivat-ads**
Changes to test connection to real CADIP and ADGS stations:

- remove unused `stationConfigFile` parameter
- fix documentation
- allow to configure OAuth2 token endpoint. It is not the same endpoint across all real CADIP stations
- remove the need to configure individual OData endpoints: all CADIP stations implement the CADIP ICD with "/Files" and "/Sessions" endpoints. Only the root OData url needs to be configured (once).
- change the session href mapping to use /Sessions endpoint instead of /Files

To use with https://github.com/RS-PYTHON/rs-server/pull/687

### CODE REVIEW

File: charts/rs-server-cadip/templates/configmap.yaml [hunk 1]
```diff
@@ -35,9 +35,9 @@ data:
 
         search:
           type: ODataV4Search
-          api_endpoint: '{{ $v.endpoint.url.file }}'
+          api_endpoint: '{{ $v.endpoint.url.odata }}/Files'
           need_auth: true
-          results_entry: responses | $
+          results_entry: $.value
```
Comment (jgaucher-cs) [inline charts/rs-server-cadip/templates/configmap.yaml]: @vprivat-ads @Padeanu in rs-server we have 4 template session files: 
1. https://github.com/RS-PYTHON/rs-server/blob/fix/rspy491-cadip-search-fix-rebased-3-matthieu/services/cadip/config/cadip_ws_config.template.yaml
2. https://github.com/RS-PYTHON/rs-server/blob/fix/rspy491-cadip-search-fix-rebased-3-matthieu/services/cadip/config/cadip_ws_config.template_session.yaml
3. https://github.com/RS-PYTHON/rs-server/blob/fix/rspy491-cadip-search-fix-rebased-3-matthieu/services/cadip/config/cadip_ws_config_token_module.template.yaml
4. https://github.com/RS-PYTHON/rs-server/blob/fix/rspy491-cadip-search-fix-rebased-3-matthieu/services/cadip/config/cadip_ws_config_token_module.template_session.yaml

Or see the two config files you edited before I have merged the templates into your branch: 
1. https://github.com/RS-PYTHON/rs-server/blob/d682b21d14402554c9320481c58929faacc34e17/services/cadip/config/cadip_ws_config.yaml#L635
2. https://github.com/RS-PYTHON/rs-server/blob/d682b21d14402554c9320481c58929faacc34e17/services/cadip/config/cadip_ws_config_token_module.yaml

I don't know well the difference between the 4 template files, but Vincent in 2 of them you left `results_entry: responses | $` and in the two others you changed to: `results_entry: $.value` 

But here in rs-helm you used `results_entry: $.value` in the 4 occurrences, so which one is right ?

Comment (vprivat-ads) [reply charts/rs-server-cadip/templates/configmap.yaml]: @jgaucher-cs  `$.value` everywhere. I probably forgot some cases

File: charts/rs-server-cadip/templates/configmap.yaml [hunk 2]
```diff
@@ -102,17 +102,28 @@ data:
             next_page_url_tpl: '{url}?{search}&$top={items_per_page}'
             total_items_nb_key_path: '$.total'
 
+          sort:
+            sort_by_tpl: '&$orderby={sort_param} {sort_order}'
+            sort_param_mapping:
+              startTimeFromAscendingNode: DownlinkStart
```
Comment (jgaucher-cs) [inline charts/rs-server-cadip/templates/configmap.yaml]: Same here @vprivat-ads, if I look at your two last changes in rs-server cadip config files: 
1. https://github.com/RS-PYTHON/rs-server/blob/d682b21d14402554c9320481c58929faacc34e17/services/cadip/config/cadip_ws_config.yaml
2. https://github.com/RS-PYTHON/rs-server/blob/d682b21d14402554c9320481c58929faacc34e17/services/cadip/config/cadip_ws_config_token_module.yaml

The 3 fields: 
```yaml
        startTimeFromAscendingNode: DownlinkStart
        completionTimeFromAscendingNode: DownlinkStop
        publicationDate: PublicationDate
```

are defined only for the _session stations. But in this helm chart, you defined them for all stations.

Comment (Padeanu) [reply charts/rs-server-cadip/templates/configmap.yaml]: This needs to be updated according to https://github.com/RS-PYTHON/rs-server/blob/b6fd7ea4c4939b80fe862b8b8bc440eb3be54653/services/cadip/config/cadip_ws_config_token_module.template.yaml#L92

Comment (vprivat-ads) [reply charts/rs-server-cadip/templates/configmap.yaml]: I think for the "non session" stations, only publicationDate should be added, there is no downlinkStart/Stop info at file level, it's only at session level

**Review by jgaucher-cs — APPROVED** [review]

**Pull request merged**